doc/sphinxman/source/build_planning.rst

```diff
@@ -1856,3 +1856,28 @@ to your dev AddOn dynamic lib and update automatically when you rebuild
 the AddOn lib. Naturally, you may need to delete ``core.so`` and remake
 as needed.
 
+
+.. _`faq:logging`:
+
+How to use logging in Psi4
+--------------------------
+
+A log file is generated alongside the output file and has extension
+``.log``. Selected key information is sent to it. The log file contents
+are provisional and haphazard, so don't start relying upon them. It is
+tentatively envisioned that log files will have much the same results as
+files but more as structured data than narrative, nicely formatted data.
+In some cases where external calls are involved, the most detailed
+information may only be collectable in the log file. It is
+worthwhile to check the log file if expected results are missing from
+the usual output file. If the log file is empty, the logging level may
+be set too low.
+
+To change the logging level to DEBUG in PSIthon::
```

Comment:
I would say:
`To change the logging level to DEBUG from the command-line::`

Reply:
Both changes applied, thanks.